tests/unit_test.py

```diff
@@ -152,15 +152,17 @@ def test_auto_install_native():
     except RuntimeWarning as w:
       rw = w
     assert rw, "Expected a RuntimeWarning from unversioned auto-install"
-    params = re.search(
+    match:Optional[re.Match] = re.search(
       "use\\("
         "\"(?P<name>.*)\", "
         "version=\"(?P<version>.*)\", "
         "hash_value=\"(?P<hash_value>.*)\", "
         "auto_install=True"
       "\\)",
       rw.args[0]
-    ).groupdict()
```

Comment:
does that work now?

Reply:
Yes, it does, it's jappy with the `version, hash_value = hit` added elsewhere in the main file